data/lavezzo2020suppression/lavezzo2020suppression-extraction.md

```diff
@@ -0,0 +1,250 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: python_clean
+    language: python
+    name: python3
+---
+
+```python
+import yaml
+import pandas as pd
+import numpy as np
+import textwrap
+import datetime
+
+
+#functions to add folded blocks and literal blocks;
+class folded_str(str): pass
+class literal_str(str): pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer);
+```
+
+```python
+#Read Excel file into the environment
+df_demo = pd.read_excel('anonymised_data_public_final.xlsx', sheet_name='anonymised_dataset')
+df_pcr = pd.read_excel('anonymised_data_public_final.xlsx', sheet_name='RT_PCR_DATA')
+# list(df_demo.columns)
+
+```
+
+```python
+# Filter the useful columns in df_demo
+df_demo2 = df_demo.loc[:,'id':'gender'].join(df_demo['first_symptoms_date'])
```

Comment:
It looks like both `df_demo` and `df_demo2` are used in the code below. Would it make sense to just replace `df_demo` with the filtered value or do we need both the filtered and unfiltered data later?

Reply:
df_demo was used in later codes so that this naming is not changed.